atlasdb-client/src/main/java/com/palantir/atlasdb/cache/OffHeapTimestampCache.java

```diff
@@ -132,8 +134,8 @@ public Long getCommitTimestampIfPresent(Long startTimestamp) {
     }
 
     private static CacheDescriptor createNamespaceAndConstructCacheProposal(
-            PersistentTimestampStore persistentTimestampStore) {
-        StoreNamespace proposal = persistentTimestampStore.createNamespace(TIMESTAMP_CACHE_NAMESPACE);
+            LogicalPersistentStore<?, ?> persistentStore) {
```

Comment:
what's with the question mark generics?

proposal/namespace/descriptor? can we agree on the terminology?

Reply:
Proposal and descriptor are internal to cache.